cms/djangoapps/contentstore/views/videos.py

```diff
@@ -147,6 +156,19 @@ def videos_handler(request, course_key_string, edx_video_id=None):
         return videos_post(course, request)
 
 
+@expect_json
+@login_required
+@require_POST
+def video_images_handler(request, course_key_string, edx_video_id=None):
+    image = request.POST['file']
+    update_video_image(edx_video_id, course_key_string, image)
+    LOGGER.info(
+        'VIDEOS: Video image updated for edx_video_id [%s]',
+        edx_video_id
+    )
+    return JsonResponse(status=201)
```

Comment:
This will have to return a thumbnail url in response as suggested by @muhammad-ammar yesterday.